snap7/client.py

```diff
@@ -344,8 +341,7 @@ def get_block_info(self, blocktype, db_number):
         if not blocktype:
             raise Snap7Exception("The blocktype parameter was invalid")
 
-        logger.debug("retrieving block info for block %s of type %s" %
-                      (db_number, blocktype))
+        logger.debug("retrieving block info for block {} of type {}".format(db_number, blocktype))
```

Comment:
It seems that `logger `doesn't support this style (even if I wish it would). Logger either use the old style with %s or .format(), so I went along with .format()

Reply:
I ran the tests and they said: 
![grafik](https://user-images.githubusercontent.com/24408657/86472612-b5dd7400-bd3f-11ea-948c-0c262207a9f4.png)

Reply:
Or may that be just the Unittests doing blackmagic here?

Reply:
Well indeed there is some black magic happening, which I can't figure out at the moment. My other PC with a quick setup of logging works like a charm with f-strings.
Mysterious, but I will deal on monday with that. If anyone may know an answer, let us know. :)

Reply:
f"" strings are converted to normal strings before being passed on to the function call, so it should be fine:
```In [1]: import logging                  

In [2]: logging.error(f"{logging}")     
ERROR:root:<module 'logging' from '/usr/lib/python3.8/logging/__init__.py'>
```

the error must be somewhere else.

Reply:
He just complains at the start/compile time with that, but later in the script he execute it without an error.
This is realy annoying and interessting at the same time. I may need a while to figure out what is happening here.

Reply:
A these I have is that my _unittest_ may be a reason on my IDE. It seems to execute python2 instead of python3 and while executing in via terminal it shows other errors, than via IDE.
If there should be a problem, than travis would have complained about it (like in the OSX test) as well I guess. Well this will be my riddle, not the repo ones so feel free to merge. All typos are fixed. ;)